packages/notifi-dataplane/lib/types/FusionMessage.ts

```diff
@@ -2,11 +2,28 @@ import { Types } from 'notifi-graphql/dist';
 
 export interface FusionMessage {
   eventTypeId: string;
-  variablesJson: any;
+  variablesJson: object | VariablesJsonPayload;
   specificWallets?: ReadonlyArray<
     Readonly<{
       walletPublicKey: string;
       walletBlockchain: Types.WalletBlockchain;
     }>
   >;
 }
+
+export type OptionalTargetType = 'Sms' | 'Telegram' | 'Discord' | 'Email';
+
+export type VariablesJsonPayload = { Platform: TargetVariables } & {
```

Comment:
Nit: As discussed, I think it's good exposing it like this. I'd suggest renaming it `CommunityManagerVariablesJsonPayload` in order to specifically indicate that this is for Community Manager posts within the code (rather than create any impression that all messages should look like this).